huaweicloud/services/cbh/data_source_huaweicloud_cbh_instance.go

```diff
@@ -237,7 +238,7 @@ func resourceCbhInstancesRead(ctx context.Context, d *schema.ResourceData, meta
 	if err != nil {
 		return diag.FromErr(err)
 	}
-	instances := flattenGetInstancesResponseBodyInstance(getCbhInstancesRespBody)
+	instances := utils.PathSearch("instance", getCbhInstancesRespBody, make([]interface{}, 0)).([]interface{})
```

Comment:
The list response value is unequal with schema, such as `instanceId`, `publicip`, `vpcID` and `subnetID`,.